src/popup/components/InputPassword.vue

```diff
@@ -17,16 +17,19 @@
     <template #label-after>
       <span
         v-if="showPasswordStrength && modelValue"
-        :class="['password-strength', passwordStrength.toLowerCase()]"
-        v-text="passwordStrength"
+        class="password-strength"
+        :class="[passwordStrength]"
```

Comment:
I suggest to prefix these classes:
```suggestion
        :class="[`strength-${passwordStrength}`]"
```

so the CSS block would look as follows:
```
&.strength {
  &-weak {
    ...
```

What do you think? In my opinion this gives some kind of feature grouping.